lib/mayaUsd/ufe/UsdContextOps.cpp

```diff
@@ -569,10 +573,23 @@ Ufe::UndoableCommand::Ptr UsdContextOps::doOpCmd(const ItemPath& itemPath)
         return std::make_shared<SetVariantSelectionUndoableCommand>(prim(), itemPath);
     } // Variant sets
     else if (itemPath[0] == kUSDToggleVisibilityItem) {
+#if UFE_PREVIEW_VERSION_NUM < 2034
+        auto attributes = Ufe::Attributes::attributes(sceneItem());
+        assert(attributes);
+        auto visibility = std::dynamic_pointer_cast<Ufe::AttributeEnumString>(
+            attributes->attribute(UsdGeomTokens->visibility));
+        assert(visibility);
+        auto current = visibility->get();
+        return visibility->setCmd(
+            current == UsdGeomTokens->invisible ? UsdGeomTokens->inherited
+                                                : UsdGeomTokens->invisible);
+#else
         auto object3d = UsdObject3d::create(fItem);
+        assert(object3d);
```

Comment:
Elsewhere we've been using TF_AXIOM, might want to consider that.

Reply:
Addressed in 8bac4732ab002bf19c0a93d5219c93385c01e74e